lib/versioning/composer/index.ts

```diff
@@ -39,6 +48,8 @@ function composer2npm(input: string): string {
   output = output.replace(/(?:^|\s)~([1-9][0-9]*(?:\.[0-9]*)?)(?: |$)/g, '^$1');
   // ~0.4 to >=0.4 <1
   output = output.replace(/(?:^|\s)~(0\.[1-9][0-9]*)(?: |$)/g, '>=$1 <1');
+  // 1.0@beta2 to 1.0-beta.2
+  output = composerStability2npm(output);
```

Comment:
@hussainweb Can this just be
```suggestion
  output = output.replace(/[@|^]/, '-');
```

Reply:
I based this on [semver docs](https://docs.npmjs.com/misc/semver#prerelease-identifiers). The prerelease identifiers are of format `beta.5`. We will need to convert `beta5` to `beta.5` as well.

Reply:
@skippednote, is `1.0.0-beta5` a valid npm specifier?